NEWS.md

```diff
@@ -34,15 +34,60 @@ The format is based on [Keep a Changelog]
 
 ## [unreleased]
 
+This is a shorter release cycle compared with the usual 10-week one because it
+contains a significant number of stability and cleanliness fixes which warrant
+packaging.
+
+We do not normally describe behind-the-scenes work in this log, but will mention
+two here as an exception:
+
+1. A clean run with gcc/clang address-sanitizer has been achieved which is a
+   great position to protect from memory leaks. Thanks to @tokyo4j for fantastic
+   work with this [#2331]
```

Comment:
The way this is described my make it seems as if address-sanitizer was never used before. Suggest removing it completely and instead use something like "A clean run with gcc/clang memory leak check has been archived [..]".